sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryUtils.java

```diff
@@ -265,6 +266,7 @@ public abstract static class Builder {
           .put(SqlTypes.DATETIME.getIdentifier(), StandardSQLTypeName.DATETIME)
           .put("SqlTimeWithLocalTzType", StandardSQLTypeName.TIME)
           .put("Enum", StandardSQLTypeName.STRING)
+          .put(MicrosInstant.IDENTIFIER, StandardSQLTypeName.TIMESTAMP)
```

Comment:
Is there a test verifying that this logical type is supported? MicrosInstant is used for `java.time.Instant` (saying, testing write `java.time.Instant` to BigQuery)

Reply:
`MicrosInstant` is also used for the translation of the Avro logical type named TimestampMicros (which is being forced in the test `BigQueryWriteIOTest.testWriteAvro()`. 
For the time related logical types (timestampmicros and millis, alongside with Date), Avro types store internally a numeric value that represents time since epoch (micros, millis or days depending on the type). 
I'm not sure if I can force an Avro object to carry a `java.time.Instant` (maybe not) and this code path executes only when the underlying Avro `GenericRecord` has a time related logical type (which is being tested) and its schema gets translated to a Beam` Row.Schema` yielding the MicrosInstant (AvroUtils.toFieldType method).

Reply:
fyi Support of Micros_instant in AvroUtils is about to rolled back due to a breaking change (#24489). By now MicrosInstant logical type (and `java.time.Instant`) support is incomplete throughout the code base. Sorry for inconvenience.

Reply:
oh, so the only supported time related logical types will be millis and date for now, is that right? 
I can remove the timestamp in micros for the time being if that works for this PR. sounds good?

Reply:
ok, in fact I rebased from master and removed the MicrosInstant mapping and the tests are still running fine. So I will keep the micros support since now those get mapped to Instant and work fine.